include/zresource.h

```diff
@@ -100,7 +100,9 @@ struct zip_service
  * }
  * @endcode
  */
-extern struct zip_service* zresource_services;
+//extern struct zip_service* zresource_services;
```

Comment:
Please go ahead and delete the old extern declaration. We will use the function call going forward.